src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift

```diff
@@ -253,15 +253,21 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
 	func coordinatorUserDidRequestReset(exposureSubmissionService: ExposureSubmissionService) {
 		exposureSubmissionService.reset()
 
-		// Reset key value store.
+		// Reset key value store. Preserve environment settings.
+		let environment = store.selectedServerEnvironment
```

Comment:
should this happen also when the reset of the database files?
Perhaps saving the environment should happen in the do-block?